### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by Sameony**
Finally completed Cards Components
image
image
image
image
image
image

### CODE REVIEW

File: Templates/Cards/cards.html [hunk 1]
```diff
@@ -2,119 +2,567 @@
...
+            <span class="ico open-ico">
+                <i class="fas fa-chevron-circle-left"></i>
+            </span>
+        </div>
+    </div>
+
+    <div class="container">
+        <h1 class="pd-y-md mg-y-sm">Cards</h1>
+        <div class="contentArticle pd-y-sm">
+            Cards are a set of similar meaning or category-falling content hoders.
```
Comment (ttahm3d) [inline Templates/Cards/cards.html]: similar meaning or category falling - this phrase doesn't sound good. Replace it with like Cards are used to display items of same type or category

**Review by nishasen — APPROVED** [review]
Looks good to me.

**Review by ttahm3d — APPROVED** [review]
small change in text. Can be merged and taken care later

File: Templates/Cards/cards.html (hunk 1 above)
Comment (Sameony) [reply Templates/Cards/cards.html]: alright, i'll see to it

**Pull request merged**